### ISSUE TITLE
Dashboards: Literal `&apos;` appears in time picker

### ISSUE THREAD

**Issue by domdfcoding** [grafana/grafana#75374]
### What happened?

In the text of a dashboard's time picker there is a literal `&apos;` where there should be an apostrophe. The full text and a screenshot are shown below:

```
It looks like you haven&apos;t used this time picker before. As soon as you enter some time intervals, recently used intervals will appear here.
Read the documentation to find out more about how to enter custom time ranges.
```

![grafana-time-panel-issue](https://github.com/grafana/grafana/assets/8050853/a2d70af6-7a12-4a6f-8cd7-ac332868173b)

### What did you expect to happen?

There is an apostrophe (i.e. it reads `It looks like you haven't used this time picker before`)

### Did this work before?

This is the first time I've noticed this and I've had Grafana installed for about a year, so I don't think this was happening before.

### How do we reproduce it?

1. Open a dashboard
2. Click on the current time range in the top panel to open the time picker

### Environment (with versions)?

Grafana: 10.1.2 (also on 10.1.1)
OS: Ubuntu 22.04
Browser: Firefox and Chrome on Ubuntu 20.04

### Grafana platform?

A package manager (APT, YUM, BREW, etc.)

**Comment by ash211** [comment]
I observed the same thing on `Grafana v10.1.3 (f79ea44a3b)`
![Screenshot 2023-09-25 at 12 52 59 PM](https://github.com/grafana/grafana/assets/357170/b2fa186b-e255-40f6-8b8e-df4a765f01ca)

**Comment by ash211** [comment]
@ifrost @aditipatelpro is this related to the conversation thread you had at https://github.com/grafana/grafana/pull/74292#discussion_r1319052406 ?

**Comment by ifrost** [comment]
> @ifrost @aditipatelpro is this related to the conversation thread you had at https://github.com/grafana/grafana/pull/74292#discussion_r1319052406 ?

Yep, I thought we addressed it 🤔 I'll have a look.

**Comment by ifrost** [comment]
Oh! It seems that #74292 actually fixed it! I think it's been happening since #58316 🤔 I can reproduce it in a version before the PR was merged and seems to be working fine in `main`.

**Comment by Elfo404** [comment]
Closing this as fixed by #74292
